### PR TITLE
ubuntu latest demo finished

### PR DESCRIPTION
**Pull request by fmontoto**
Adds a new demo using latest ubuntu distribution and new knot commands. It's much faster to build and run than the previous one, as it does not compile any dependency.

### CODE REVIEW

**Comment by Zeeerooo** [comment]
LGTM

File: tchsm/demo/ubuntulatest-knot/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## Run the Demo
+
+There is an included script to build and run the demo.
+
+To build, execute: ```demo.sh build```
+
+To start the demo: ```demo.sh start```
+
+To stop the demo: ```demo.sh stop```
+
```
Comment (camilog) [inline tchsm/demo/ubuntulatest-knot/README.md:24]: maybe add a '$' prompt before the commands

**Comment by fmontoto** [comment]
@camilog done

**Comment by camilog** [comment]
LGTM

**Pull request merged**